spring-cloud-dataflow-app-launcher/pom.xml

```diff
@@ -0,0 +1,155 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<artifactId>spring-cloud-dataflow-app-launcher</artifactId>
+	<packaging>jar</packaging>
+	<name>spring-cloud-dataflow-app-launcher</name>
+	<description>Application for launching spring-cloud-stream modules</description>
```

Comment:
should replace "modules" here, but might want to change "Application" as well (maybe "Bootstrap")